server/src/main/java/io/javaclasses/filehub/storage/tokenStorage/LoggedIdUserRecord.java

```diff
@@ -0,0 +1,47 @@
+package io.javaclasses.filehub.storage.tokenStorage;
+
+import com.google.errorprone.annotations.Immutable;
+import io.javaclasses.filehub.storage.Record;
+import io.javaclasses.filehub.storage.userStorage.UserId;
+
+import java.time.LocalDate;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Data structure for saving token in the {@link LoggedInUsersStorage}.
```

Comment:
```suggestion
 * A data structure for saving token. A unit of data of {@link LoggedInUsersStorage}.
```
Try splitting this sentence into two: "What is it?" and "What it does?" and thus you will improve readability.
- [ ] Ok.